javalib/src/main/scala/java/util/zip/CRC32.scala

```diff
@@ -27,7 +27,7 @@ class CRC32 extends Checksum {
   def update(buf: Array[Byte], off: Int, nbytes: Int): Unit = {
     if (nbytes == 0) {
       ()
-    } else if (off <= buf.length && nbytes >= 0 && off >= 0 && buf.length - off >= nbytes) { // avoid int overflow, check null buf
+    } else if (off <= buf.length && nbytes > 0 && off >= 0 && buf.length - off >= nbytes) { // avoid int overflow, check null buf
```

Comment:
If you agree, I think we should just remove the comment - I don't think it is helping to make the code more readable.

Reply:
```suggestion
    } else if (off <= buf.length && nbytes > 0 && off >= 0 && buf.length - off >= nbytes) {
```